src/cryptoadvance/specter/controller.py

```diff
@@ -243,6 +243,26 @@ def new_wallet_simple():
                 return render_template("base.html", error="Key not found", specter=app.specter, rand=rand)
             # create a wallet here
             wallet = app.specter.wallets.create_simple(wallet_name, wallet_type, key, device)
+            rescan_blockchain = 'rescanblockchain' in request.form
+            if rescan_blockchain:
+                if app.specter.info['chain'] == "main":
+                    if not app.specter.info['pruned'] or app.specter.info['pruneheight'] < 481824:
+                        startblock = 481824
+                    else:
+                        startblock = app.specter.info['pruneheight']
```

Comment:
if prune is false you will get error here because pruneheight is not in the info